app/templates/components/hopper-frontend.hbs

```diff
@@ -0,0 +1,82 @@
+<div id="hopper" class="row">
+
+    <div id="hopper-first-col" {{bind-attr class=":hopper-col :columns isSidenavOpen:medium-13:medium-15"}}>
+        <div class="hopper-content">
+            <div class="hopper-meta">
+                <div class="form-title">
+                    <h1>My awesome Festival</h1>
+                </div>
+                <div class="form-meta-toggl">
+                    <div class="switch round">
+                        <input id="exampleCheckboxSwitch" type="checkbox">
+                        <label for="exampleCheckboxSwitch"></label>
+                    </div>
+                </div>
```

Comment:
IMO a toggle switch is not what we need here. The purpose of the element is to switch between two different views, but a toggle switch is used to switch something on or off. IMO a [button group](http://foundation.zurb.com/docs/components/button_groups.html) is the right thing to use.